src/style.css

```
@@ -1 +1,13 @@
 /* styles go here */
+
+.inputForm > :not(:nth-child(1)) {
+  margin-top: 20px;
+}
```

don't increase specificity if it's unnecessary, also you can use `:first-child` instead of `:nth-child(1)`
```suggestion
.fieldsetBlock:not(:first-child) {
  margin-top: 20px;
}
```